docs/user-guide/release-notes/2.8.x.rst

```diff
@@ -2,6 +2,165 @@
+
+Additionally, CVE-2016-3095 and CVE-2016-3106 made it possible for local attackers to read the CA
+key during generation (which happens during the initial installation of Pulp or any time an admin
+ran ``pulp-gen-ca-certificate``). If you are concerned that a local user may have read that CA key
+during the brief window that it was visible it is recommended that you regenerate the key and cert.
+
+To regenerate the certificate, you should remove the old one and then you may use the provided
+utility::
+
+    # First remove the old files so that the new files get the correct SELinux context.
+    $ sudo rm /etc/pki/pulp/ca.*
+    $ sudo pulp-gen-ca-certificate
+
+If you choose not to perform the CA regeneration, you may wish to apply the correct SELinux type to
+your existing CA files as versions of Pulp < 2.8.3 generated this file with an incorrect SELinux
+type. You don't need to do this if you removed the old file and regenerated it with
+``pulp-gen-ca-certificate``. You can run restorecon recursively on the /etc/pki/pulp folder to fix
+the SELinux label on your existing CA certificate::
+
+    # You only need to do this if you didn't regenerate the CA above.
+    $ sudo restorecon -R /etc/pki/pulp
+
+
+CVE-2016-3107 and CVE-2016-3108
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For Nodes users, the /etc/pki/pulp/nodes/node.crt file was installed world-readable. Users are
+recommended to remove this file and regenerate it by running pulp-gen-nodes-certificate::
+
+    # It is important to remove the file so that the new file has the correct permissions.
+    $ sudo rm /etc/pki/pulp/nodes/node.crt
+    $ sudo pulp-gen-nodes-certificate
+
+
+CVE-2016-3111
+^^^^^^^^^^^^^
+
+Both the RSA key pair for the Pulp server and RSA key pair for each Pulp consumer was generated
+during installation in an insecure directory. This vulnerability allowed a local attacker to read
+the private key portion of the key pair. These keys are used for message authentication between the
+Pulp server and the Pulp consumers. If you are concerned that a local attacker was able to read
+these keys, you can regenerate them. We do not ship a script to perform this, but the process is
+straight-forward. For the Pulp server, do the following as root::
+
+    $ cd /etc/pki/pulp/
+    $ rm rsa.key rsa_pub.key
+    $ umask 077
+    $ openssl genrsa -out rsa.key <bits>  # <bits> should be at least 2048
+    $ openssl rsa -in rsa.key -pubout > rsa_pub.key
+    $ chgrp apache rsa.key rsa_pub.key
+    $ chmod 640 rsa.key  # Apache must be able to read the private key
+    $ chmod 644 rsa_pub.key  # The public key is world-readable as it is served via Apache
+
+The Pulp consumer key is similar::
+
+    $ cd /etc/pki/pulp/consumer/
+    $ rm rsa.key rsa_pub.key
+    $ umask 077
+    $ openssl genrsa -out rsa.key <bits>  # <bits> should be at least 2048
+    $ openssl rsa -in rsa.key -pubout > rsa_pub.key
```

Comment:
@jortel and/or @jeremycline: Can one or both of you confirm whether need the above three lines? Will the pulp-consumer register statement below will take care of this for us?